internal/distributed/connection_manager_test.go

```diff
@@ -0,0 +1,202 @@
+package distributed
+
+import (
+	"context"
+	"net"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/milvus-io/milvus/internal/log"
+	"github.com/milvus-io/milvus/internal/proto/datapb"
+	"github.com/milvus-io/milvus/internal/proto/indexpb"
+	"github.com/milvus-io/milvus/internal/proto/querypb"
+	"github.com/milvus-io/milvus/internal/proto/rootcoordpb"
+	"github.com/milvus-io/milvus/internal/util/paramtable"
+	"github.com/milvus-io/milvus/internal/util/sessionutil"
+	"github.com/milvus-io/milvus/internal/util/typeutil"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+)
+
+var Params paramtable.BaseTable
+
+func TestConnectionManager(t *testing.T) {
+	Params.Init()
```

Comment:
need more test on the timeout cases.
1) If timeout works for all cases?
2) what happens if timeout?